gradle.properties

```diff
@@ -1,31 +1,35 @@
 # IntelliJ Platform Artifacts Repositories
 # -> https://plugins.jetbrains.com/docs/intellij/intellij-artifacts.html
-
-pluginGroup = intellij
-pluginName = intellij-buf
+pluginGroup=intellij
+pluginName=intellij-buf
 # SemVer format -> https://semver.org
-pluginVersion = 0.1.3
-
+pluginVersion=0.1.3
```

Comment:
should we bump this to `1.6.0`, or after doing this? https://github.com/JetBrains/intellij-platform-plugin-template/blob/main/gradle.properties#L7

Reply:
This is our version - https://plugins.jetbrains.com/plugin/19147-buf-for-protocol-buffers/versions. I'll bump to 0.1.4 when we're ready to release something.

Reply:
ah yep, definitely missed that.